src/strings.json

```diff
@@ -25,8 +25,8 @@
   "watchExpressions.header": "Watch Expressions",
   "watchExpressions.addButton": "Add Watch Expression",
   "watchExpressions.refreshButton": "Refresh",
-  "welcome.search": "%S to search for files",
-  "sourceSearch.search": "Search...",
```

Comment:
yeah, the properties file is a gotcha :)

https://github.com/devtools-html/debugger.html/pull/1482